corehq/apps/users/middleware.py

```diff
@@ -77,14 +77,23 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         ):
             return None
 
-        if not toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(request.user.username):
-            return None
-        elif request.user.is_staff or request.user.is_superuser and not request.user.is_verified():
-            if request.path.startswith('/account/') or request.couch_user.two_factor_disabled:
-                return None
+        return self.enforce_two_factor(toggles, request)
+
+    def enforce_two_factor(self, toggles, request):
+        if toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(request.user.username):
```

Comment:
Pet peeve: Instead of nesting these, could you swap the condition and return one by one.

ex. 
```
if not toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(request.user.username):
    return None
if request.user.is_verified():
    return None
```
Just makes it a bit more readable

Reply:
I agree. You can also put them all in one statement
```py
if request.couch_user.two_factor_disabled or request.user.is_verified() or request.path.startswith('/account'):
    return None
return TemplateResponse(...)
```

Reply:
Done: Put all the if statements in one line, put the TemplateResponse first so I could start with a positive if (which I think is slightly less confusing)